docs/transitions.md

```diff
@@ -0,0 +1,2 @@
+# Transitions
+The transition change state of object from one to another - that's helps in product lifecycle manegment. For each object (asset, support, licence) you can define some workflow (set of transitions) and provide special actions for each transitions.
```

Comment:
Add an example as you would read that kind of documentation. Just think about it.

Chaging object from one state to another could mean anything, including changing a state of single atom spin, during the reading of this documentation.